docs/interactions/Slash_Commands.md

```diff
@@ -790,21 +792,24 @@ If you specify `choices` for an option, they are the **only** valid values for a
 
 ## Interaction
 
-An interaction is the base "thing" that is sent when a user invokes a command, and is the same for Slash Commands and other future interaction types
+An interaction is the base "thing" that is sent when a user invokes a command, and is the same for Slash Commands and other future interaction types.
 
-| Field      | Type                                                             | Description                                                    |
-|------------|------------------------------------------------------------------|----------------------------------------------------------------|
-| id         | snowflake                                                        | id of the interaction                                          |
-| type       | InteractionType                                                  | the type of interaction                                        |
-| data?\*    | ApplicationCommandInteractionData                                | the command data payload                                       |
-| guild_id   | snowflake                                                        | the guild it was sent from                                     |
-| channel_id | snowflake                                                        | the channel it was sent from                                   |
-| member     | [guild member](#DOCS_RESOURCES_GUILD/guild-member-object) object | guild member data for the invoking user, including permissions |
-| token      | string                                                           | a continuation token for responding to the interaction         |
-| version    | int                                                              | read-only property, always `1`                                 |
+| Field        | Type                                                             | Description                                                    |
+|--------------|------------------------------------------------------------------|----------------------------------------------------------------|
+| id           | snowflake                                                        | id of the interaction                                          |
+| type         | InteractionType                                                  | the type of interaction                                        |
+| data?\*      | ApplicationCommandInteractionData                                | the command data payload                                       |
+| guild_id?    | snowflake                                                        | the guild it was sent from                                     |
+| channel_id?  | snowflake                                                        | the channel it was sent from                                   |
```

Comment:
In what case(s) is channel_id going to not be included,?

Reply:
In all existing use cases, you will get a channel ID. This is more of a future-proofing thing. For example, if we allowed bots to add options to context menus, you may be able to add an interaction by right clicking on a guild or a user, in which case that _might_ not include a channel ID.

I say might not because none of that stuff exists yet, so again this is more future-proofing. In both guild commands and DM commands, you will get a channel id.